urlchecker/main/github.py

```diff
@@ -58,7 +62,7 @@ def delete_repo(base_path):
     return result.returncode
 
 
-def get_branch(default="master"):
+def get_branch(default: str = "master") -> str:
```

Comment:
Maybe we should update this to main? Enough have changed that it might be about the right time. If you don't want to do for this PR don't worry about it, just a suggestion now that I see it here!
```suggestion
def get_branch(default: str = "main") -> str:
```

Reply:
that is a good point. I think this is present in other places in the code

Reply:
We can keep as is for now - and then change all the other places at once in a more scoped PR. Just wanted to bring it up for discussion!

Reply:
I think we had this discussion before. I am all up for it. I just made the changes but changing master to main in https://github.com/urlstechie/urlchecker-python/blob/3a13e4ecff17ea23a83b86c43affec1f8e1bcebb/urlchecker/main/github.py#L16 breaks the tests so I am leaving that a different PR.  The other changes seem to work just fine, should I keep them?

Reply:
Yes the other changes are good!